docs/overview/versioning-policy.md

```diff
@@ -32,7 +32,7 @@ their own release cadence and versioning policy.
 
 ## Main Release Line
 
-Release cadence: Monthly
+Release cadence: Monthly, specifically on the third Tuesday of each month. The first release took place in March 2022.
```

Comment:
It's currently the Tuesday before the third Wednesday of each month :sweat_smile: 

The reason for that is that we wanted it to be just before the community session.

Reply:
Done! Thanks @Rugvip